elastic_transport/_node/_http_urllib3.py

```diff
@@ -199,7 +199,7 @@ def perform_request(
                 body=body,
                 exception=err,
             )
-            raise err from None
+            raise err from e
```

Comment:
This is also unrelated, but it feels wrong to hide parts of exceptions. (And this was needed to debug chain certs code.)